### PR TITLE
Create a recommendation for aoharris-hopkinton file 

### PR DESCRIPTION
**Pull request by aoharris**
Things to do:

- [x] add commit
- [x] Review comment
- [x] Official review
- [x]  Merge

### CODE REVIEW

**Comment by ghost** [comment]
This is a comment...

File: aoharris-hopkinton.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+### Places to eat
+- Framingham EONOODLES
+- House Of China
+
+### Places to Visit
+- Boston Common
+- Franklyn Zoo
+
+### Places to Avoid
+- Syria
```
Comment (ghost) [inline aoharris-hopkinton.md]: 😨

**Review by ghost — APPROVED** [review]
Looks good.

**Review by aoharris — COMMENTED** [review]
Approve changes.

**Comment by aoharris** [comment]
Merging changes

**Pull request merged**